### ISSUE TITLE
marginLeft/marginRight/marginStart/marginEnd is backwards when direction == RTL

### ISSUE THREAD

**Issue by kmelmon** [microsoft/react-native-windows#4645]
These properties have the opposite effect of iOS in RTL:
marginLeft
marginRight
marginStart
marginEnd

Min Repro:

https://snack.expo.io/vKVqJmmvB

Run on iOS and notice the margin is applied to the right side.

On Windows, notice the margin is applied to the left side instead of the right side

**Comment by kmelmon** [comment]
I'll take this as I'm fixing several RTL related issues

**Comment by acoates-ms** [comment]
Just in case you are assuming that marginLeft should flip in RTL, in RN left always means left.  If you want a margin that flips you would use marginStart.

So in your sample I would expect the margin to be on the left side.

**Comment by kmelmon** [comment]
@acoates-ms all of my observations are based on testing on iOS in an Expo Snack.  In this testing I'm not seeing any difference between using properties with "left" in the name vs "start" in the name.  Both apply to the left side in LTR, and the right side in RTL.  The RN docs don't actually say what the expected behavior actually is so I'm going off of what I'm seeing in actual testing.

**Comment by kmelmon** [comment]
I asked the question on Discord, here's the response I got from Eli:

@kmelmon, I believe this is the known behavior. RN has always flipped left and right in RTL mode. To enable people to have proper RTL support the start and end properties were introduced, but we've never found a good way to migrate the ecosystem to get the left/right properties to not flip anymore. There is a command I18nManager.swapLeftAndRightInRTL(false) that can disable that flipping. Eventually that should be the default, but it is such a big breaking change we have punted on it for 3 years (timing based on some internal documents I've found)
